tests/robot-cases/Group11-Nightly/LDAP.robot

```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
     Sign In Harbor  ${HARBOR_URL}  mike  zhu88jie
     Close Browser
 
+Test Case - Add A Never Login Member 
```

Comment:
Test Case - Sys Admin On-board New Member

and add one more case to on-board member via project admin.